### ISSUE TITLE
Databend for OLTP web applications.

### ISSUE THREAD

**Issue by ansarizafar** [datafuselabs/databend#3346]
Is there a plan to make databend suitable for OLTP web applications?

**Comment by BohuTANG** [comment]
> Is there a plan to make databend suitable for OLTP web applications?

Hi,

What's the `OLTP web applications` requirements?
If it's data-intensive applications(OLAP-based) i think it's no problem.

**Comment by Xuanwo** [comment]
@BohuTANG Let's move question alike issues to the [discussion](https://github.com/datafuselabs/databend/discussions) Q&A instead.

**Comment by Xuanwo** [comment]
I think all issues under `C-quesion` should be moved either.

Here is the query: https://github.com/datafuselabs/databend/issues?q=is%3Aopen+is%3Aissue+label%3AC-question

**Comment by ansarizafar** [comment]
Currently, databend can't be used to update or delete rows. Documentation shows only select and insert statement under data manipulation language section.

Let me rephrase my question, Is it possible to replace mysql or postgres with databend for using as a back end data store for an eCommerce or ERP web application?
